styles/fonts.css

```diff
@@ -0,0 +1,39 @@
+@font-face {
+	font-family: 'montserrat';
+  src:
+    url('/fonts/Montserrat-Regular.ttf') format('truetype'),
+    url('/fonts/montserrat-regular-webfont.woff') format('woff'),
+    url('/fonts/montserrat-regular-webfont.woff2') format('woff2');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src: 
+    url('/fonts/Montserrat-Medium.ttf') format('truetype'),
+    url('/fonts/montserrat-medium-webfont.woff') format('woff'), 
+    url('/fonts/montserrat-medium-webfont.woff2') format('woff2');
+	font-weight: 500;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src: 
+    url('/fonts/Montserrat-SemiBold.ttf') format('truetype'),
+    url('/fonts/montserrat-semibold-webfont.woff') format('woff'),
+    url('/fonts/montserrat-semibold-webfont.woff2') format('woff2');
+	font-weight: 600;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'montserrat';
+  src:
+    url('/fonts/Montserrat-Bold.ttf') format('truetype'),
+    url('/fonts/montserrat-bold-webfont.woff') format('woff'),
+    url('/fonts/montserrat-bold-webfont.woff2') format('woff2');
+	font-weight: bold;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla_Privezentsev/pullRequest?prid=5491648)